tools/export/vscode/tasks.tmpl

```diff
@@ -2,21 +2,34 @@
     // See https://go.microsoft.com/fwlink/?LinkId=733558
     // for the documentation about the tasks.json format
     "version": "0.1.0",
-    "name": "make",
-    "command": "make",
     "isShellCommand": true,
-    "args": ["-j"],
-    "showOutput": "always",
-    "problemMatcher": {
-        "owner": "cpp",
-        "fileLocation": ["relative", "${workspaceRoot}/mbed-os"],
-        "pattern": {
-            "regexp": "^(.*):(\\d+):(\\d+):\\s+(warning|error):\\s+(.*)$",
-            "file": 1,
-            "line": 2,
-            "column": 3,
-            "severity": 4,
-            "message": 5
+    "showOutput": "always",,
+    "tasks": [
+        {
+            "taskName": "make",
+            "problemMatcher": {
+                "owner": "cpp",
+                "fileLocation": ["relative", "${workspaceRoot}/mbed-os"],
+                "pattern": {
+                    "regexp": "^(.*):(\\d+):(\\d+):\\s+(warning|error):\\s+(.*)$",
+                    "file": 1,
+                    "line": 2,
+                    "column": 3,
+                    "severity": 4,
+                    "message": 5
+                }
+            }
         }
+    ],
+    "linux": {
+        "command": "make",
+        "args": ["-j"]
+    },
+    "osx": {
+        "command": "make",
+        "args": ["-j"]
+    },
+    "windows": {
+        "command": "${workspaceRoot}\\.vscode\\bin\\windows\\make.bat"
```

Comment:
@theotherjimmy It sets the PATH first to point to the cross-compilation toolchain, and then just invokes GNU Make (with `-j`).